### ISSUE TITLE
[Doc]: Line2D.set_alpha is not documented

### ISSUE THREAD

**Issue by jorgman1** [matplotlib/matplotlib#25965]
### Documentation Link

https://matplotlib.org/stable/api/_as_gen/matplotlib.lines.Line2D.html#matplotlib.lines.Line2D

### Problem

* set_alpha(alpha) is not documented

**Comment by QuLogic** [comment]
`set_alpha` is documented in the `Artist` base class.

**Comment by NikosNikolaidis02** [comment]
> `set_alpha` is documented in the `Artist` base class.
So what does it need to be closed?

**Comment by timhoffm** [comment]
I'm tempted to close this. While one can set `:inherited-members:` in `.. autoclass:` (https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directives) this would really clutter the docs. IMHO sphinx is not well suited to document class hierarchies.

**Comment by tacaswell** [comment]
I had a version of @timhoffm 's comment typed out (but unposted 🎉 for multi-tasking an forgetting to hit "Comment").

Given that I agree I am going to go ahead and close this.   Thank you for reporting this @jorgman1 but we are going to close this with no-action as if we propagated one of the inherented methods we should propagate them all on all of the `Artist` sub-classes and, as Tim points out, that would make the docs quite cluttered. 

For reference : https://matplotlib.org/stable/api/artist_api.html#matplotlib.artist.Artist is the base class docs where we have manually sorted the (many) `Artist` methods.
